<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkFiducialRegistrationWidget</class>
 <widget class="QWidget" name="QmitkFiducialRegistrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QRadioButton" name="m_rbStaticRegistration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>static Fiducial Registration</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="m_rbContinousRegistration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>hybrid continuous  Fiducial Registration</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="registrationGroupBox">
     <property name="title">
      <string>Select fiducials in image and OR (world)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="sourceLandmarksGroupBox">
        <property name="title">
         <string> Image fiducials</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QmitkPointListWidget" name="m_RegistrationImagePoints">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="m_AddImageFiducialBtn">
           <property name="text">
            <string>Add image fiducial</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="targetLandmarksGroupBox">
        <property name="title">
         <string>OR fiducials </string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QmitkPointListWidget" name="m_RegistrationTrackingPoints"/>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="m_AddTrackingFiducialBtn">
           <property name="text">
            <string>&amp;Add current 
instrument position</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="m_RegisterFiducialsBtn">
     <property name="text">
      <string>Execute Fiducial Registration</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="m_UseICPRegistration">
     <property name="text">
      <string>Find fiducial correspondences (needs 6+ fiducial pairs)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="m_RegistrationQualityDisplay">
     <property name="text">
      <string>Registration still pending...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QListWidget</extends>
   <header>QmitkPointListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
